src/components/NotesMindMap/NotesMindMapSagas.js

```diff
@@ -109,8 +109,8 @@ function* searchNoteSaga({ data }) {
   console.log('Search results: ', results);
 }
 
-function didNotAttemptToFetchChildren(note) {
-  return (!note.children || !note.children.length) && !note.hasNoChildren;
+function didNotAttemptToFetchChildren(note, edges) {
+  return !(edges ? edges.filter(edge => edge.from === note.id ).length > 0 : false);
```

Comment:
It looks like `edges` are initialized as `[]` in `selectRootNote`. `Boolean([])` is `true`.

So will `Boolean(edges)` ever be `false`, do we need to handle this situation?